### ISSUE TITLE
About the training of MURA

### ISSUE THREAD

**Issue by Smiling-Weeping-zhr** [suinleelab/vit-shapley#15]
Dear authors, I've been following up on your project recently, and I'd like to ask you about the training of the explainer in MURA, is it normal for the loss to drop to 0 after about 10 rounds of training.
Thank you very much for your answers.
![image](https://github.com/user-attachments/assets/cfd80c7f-44bc-4151-9e58-96729ec9e616)

**Comment by chanwkimlab** [comment]
Hi, it's not normal for the loss to drop to 0. The ViT Shapley loss function should continue decreasing as training goes on.
